### PR TITLE
Created log files.

### PR DESCRIPTION
**Pull request by LorenzoBaesso**

### CODE REVIEW

File: infrastructure/beacon_server.py [hunk 1]
```diff
@@ -352,6 +355,17 @@ def main():
...
         sys.exit()
+
+    isd_id = str(beacon_server.topology.isd_id)
+    ad_id = str(beacon_server.topology.ad_id)
+    ip_addr = str(beacon_server.addr)
+    log_file = '/'.join(['../logs', str(datetime.date.today()), 'ISD' + isd_id,
+        'AD' + ad_id, 'bs' + isd_id + '-' + ad_id + '-' + ip_addr + '.log'])
+    if not os.path.exists(os.path.dirname(log_file)):
+        os.makedirs(os.path.dirname(log_file))
+    logging.getLogger('').handlers = []
```
Comment (shitz) [inline infrastructure/beacon_server.py:366]: Why is this needed?

File: infrastructure/beacon_server.py [hunk 2]
```diff
@@ -352,6 +355,17 @@ def main():
...
+    logging.basicConfig(filename=log_file, filemode='w', level=logging.DEBUG)
```
Comment (shitz) [inline infrastructure/beacon_server.py:367]: `filemode` should be 'a' (append) otherwise the logs will always be overwritten.

File: infrastructure/beacon_server.py (hunk 1 above)
Comment (LorenzoBaesso) [inline infrastructure/beacon_server.py:366]: Clearing the handlers is needed when the logging facility has already been configured (see above). And we can't remove the first basicConfig because logging is used in the server inits, and therefore automatically configured with the default settings.

File: infrastructure/beacon_server.py (hunk 2 above)
Comment (LorenzoBaesso) [inline infrastructure/beacon_server.py:367]: That's exactly what I think it's best to have because after a few seconds we run SCION there's already a number of MBs more because of the logs. Plus I think it's confusing in the debugging scenario to have logs from previous runs.

Comment (shitz) [inline infrastructure/beacon_server.py:367]: I agree with the debugging scenario, but otherwise this is not really like logs work...
